lib/elixir_sense.ex

```diff
@@ -372,21 +375,32 @@ defmodule ElixirSense do
   """
   @spec references(String.t(), pos_integer, pos_integer) :: [References.reference_info()]
   def references(code, line, column) do
-    {subject, {line, col}} = Source.subject_with_position(code, line, column)
-
-    buffer_file_metadata = Parser.parse_string(code, true, true, line)
-
-    %State.Env{
-      imports: imports,
-      aliases: aliases,
-      module: module,
-      scope: scope,
-      scope_id: scope_id
-    } = Metadata.get_env(buffer_file_metadata, line)
-
-    vars = buffer_file_metadata.vars_info_per_scope_id[scope_id] |> Map.values()
-    arity = Metadata.get_call_arity(buffer_file_metadata, line, col)
-
-    References.find(subject, arity, imports, aliases, module, scope, vars)
+    with {subject, {line, col}} <- Source.subject_with_position(code, line, column) do
```

Comment:
I would just make this a case since it's a single clause. There's a recent Credo rule about that.